internal/services/healthcare/healthcare_dicom_resource.go

```diff
@@ -239,10 +370,38 @@ func resourceHealthcareApisDicomServiceUpdate(d *pluginsdk.ResourceData, meta in
 		return fmt.Errorf("expanding `identity`: %+v", err)
 	}
 
+	var enableDataPartitions *bool
+	if v, ok := d.GetOk("data_partitions_enabled"); ok {
+		enableDataPartitions = pointer.To(v.(bool))
+	}
+
+	var corsConfiguration *dicomservices.CorsConfiguration
+	if v, ok := d.GetOk("cors"); ok {
+		corsConfiguration = expandDicomServiceCorsConfiguration(v.([]interface{}))
+	}
+
+	var encryption *dicomservices.Encryption
+	if v, ok := d.GetOk("encryption_key_url"); ok {
+		encryption = &dicomservices.Encryption{
+			CustomerManagedKeyEncryption: &dicomservices.EncryptionCustomerManagedKeyEncryption{
+				KeyEncryptionKeyUrl: pointer.To(v.(string)),
+			},
+		}
+	}
+
+	var storageConfiguration *dicomservices.StorageConfiguration
+	if v, ok := d.GetOk("storage"); ok {
+		storageConfiguration = expandStorageConfiguration(v.([]interface{}))
+	}
```

Comment:
This is also ForceNew so shouldn't be updateable

Reply:
Removed from the update function